docs/scos/user/technology-partners/202204.0/payment-partners/unzer/install-and-configuration-unzer.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Installing and configuring Unzer.
+description: Install and configure Unzer module to work in the Spryker Commerce OS.
+last_updated: Jun 07, 2022
+template: concept-topic-template
+originalLink: 
+originalArticleId:
+related:
+- title: Unzer integration guide
+  link: docs/scos/dev/technology-partner-guides/page.version/payment-partners/unzer/project-integration-unzer.html
+---
+
+This topic describes how to install and configure the Unzer module to integrate Unzer into your project.
+
+## Integration overview
+
+The `spryker-eco/unzer` module provides integration of Spryker e-commerce system with Unzer technology partner. It requires the `SprykerEco.UnzerApi` `spryker-eco/unzer-api` module that provides the REST Client for making API calls to the Unzer Payment Provider.
+It also requires `SprykerEco.UnzerGui` `spryker-eco/unzer-gui` module that provides Backoffice functionality to configure Unzer credentials.
```

Comment:
why do we duplicate module naming here using different approaches?

> `SprykerEco.UnzerGui` `spryker-eco/unzer-gui`
> `SprykerEco.UnzerApi` `spryker-eco/unzer-api`

Reply:
What do you mean @DmitryLymarenko ?